README.rst

```diff
@@ -101,17 +101,6 @@ Protobuf + gRCP  ``make gen-protobuf``
 All              ``make run-generators``
 ================ =======================
 
-Docs Generation
```

Comment:
README.md in root is generated from `docs`, so you have to make the edit in the particular source file (which is `docs/pages/development/1-development.rst`), then run `make gen-readme` from repo root.

Reply:
We have too many useless dependencies. 😅